src/js/components/TaskDetail.js

```diff
@@ -287,37 +353,66 @@ class TaskSidePanelContents extends SidePanelContents {
     super.tabs_handleTabClick(...arguments);
   }
 
+  getNotFound(itemType) {
+    return (
+      <div className="container container-fluid container-pod text-align-center">
+        <h3 className="flush-top text-align-center">
+          {`Error finding ${itemType}`}
+        </h3>
+        <p className="flush">
+          {`Did not find a ${itemType} by the id "${this.props.params.taskID}"`}
```

Comment:
To expand on my comment above, relying on `this.props.params.taskID` makes this generic error node far less generic.